kubelet/datadog_checks/kubelet/common.py

```diff
@@ -21,6 +21,13 @@ def is_excluded(name, image):
 
 CADVISOR_DEFAULT_PORT = 0
 
+def urljoin(*args):
+    """
+    Joins given arguments into an url. Trailing but not leading slashes are
+    stripped for each argument.
+    :return: string
+    """
+    return "".join(map(lambda x: str(x).rstrip('/'), args))
```

Comment:
That sounds like a bit dangerous if the 2nd part doesn't have a leading slash. What about:

`return urljoin(endpoint, path.lstrip("/")) ?`

Reply:
indeed, it's better.
```
>>> urljoin("https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/proxy", "/pods")
'https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/proxy/pods'
>>> urljoin("https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/proxy", "pods")
'https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/proxypods'

>>> from six.moves.urllib.parse import urljoin
>>> urljoin("https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/proxy", "pods".lstrip("/"))
'https://10.100.0.1:443/api/v1/nodes/fargate-ip-192-168-250-49.us-east-2.compute.internal/pods'
```

Reply:
How about `return '/'.join(map(lambda x: x.strip('/'), args))`?